docs/proposals/multi-version.md

```diff
@@ -0,0 +1,128 @@
+# Multi-Version-Landscaper-Instances
+
+This document contains a proposal how to support multiple versions of Landscaper Instances in one Landscaper as a 
+Service (LaaS) landscape.
+
+## Motivation
+
+A LaaS landscape should support different versions of a Landscaper instance (LI), i.e. different LandscaperDeployments
+install different versions of a [this](../../.landscaper/landscaper-instance) component. A LaaS landscape should support 
+a predefined set of LI versions, which change from release to release (probably if the LaaS component version changes)
+and a customer could select when to upgrade which of his instances.
+
+A LI is a complex software artefact used by the customers for their deployments. Upgrading a LI might include complex
+things like updating the used Helm version, the kubernetes version of the Landscaper resource shoot cluster and many
+more. Due to this complexity, it is impossible to ensure that LI updates might not harm an existing customer deployments.
+
+To reduce this risk, customers must have possibility to test a new LI component version with their development setup 
+first, before they decide to upgrade the LI component of their productive environment.
+
+## Requirements
+
+This chapter defines the rules for supported LI versions in a LaaS landscape:
+
+- Versioning:
+  - Patch: A new patch version of a LI should only include bugfixes and security fixes. It should not break a customer
```

Comment:
```suggestion
  - Patch: A new patch version of a LI should only include bugfixes and security fixes. It must not break a customer
```

A patch release that would break a customer LI should be considere a bug and must be avoided. I think we should use strong wording here.

Reply:
I have changed it that way